src/mesh.cpp

```diff
@@ -467,6 +474,10 @@ scene::IMesh* convertNodeboxNodeToMesh(ContentFeatures *f)
 			// front
 			tx1, 1-ty2, tx2, 1-ty1,
 		};
+
+		if (use_default_txc)
+			txc = txc_default;
```

Comment:
This is inconsistent with your other single line if statement bodies, there aren't any braces.
Make up your mind.  Make all one-line if statements with or without brackets, stop with this mixture of both.